src/ui/GUIApplication.java

```diff
@@ -51,16 +51,9 @@ public void start(Stage stage) throws Exception {
 	}
 	
 	//High Scores screen
-	//@Override
-	public void start(Scene highScoresScene) throws Exception {
+	@Override
```

Comment:
Deze `@Override` moet weg en de `throws Exception` mag volgens mij ook weg.

Test altijd je code nadat je er iets aan verandert. `@Override` kan alleen gebruikt worden op een method van een super-class. `setupHighScoresScene` is dat niet en de `@Override` is dus fout en zorgt ervoor dat je de code niet kan runnen. Dit soort errors zijn niet zo moeilijk te vinden, maar daarvoor moet je wel je code testen.